test/repl.jl

```diff
@@ -397,7 +431,6 @@ temp_pkg_dir() do project_path; cd(project_path) do
         @test "remove" in c
         c, r = test_complete("help r")
         @test "remove" in c
-        @test !("rm" in c)
```

Comment:
Why remove this test?

Reply:
Because `rm` is a valid command. Not sure why this was not allowed before?

Reply:
Because we didn't want to show both the short and long form of the same command in the completion.

Reply:
K, I can bring that behavior back if needed. Note that `command_declarations` currently does not differentiate between long and short forms. It just accepts a list of names which refer to the same command. Perhaps that is too flexible?

Reply:
I think it is enough with one canonical (long) form and an optional short form.

Reply:
Sounds good. I'll change it